quinn-proto/src/connection/streams.rs

```diff
@@ -350,7 +350,12 @@ impl Streams {
             return Ok(ShouldTransmit::new(false));
         }
         let bytes_read = rs.assembler.bytes_read();
-        self.on_stream_frame(true, id);
+        let stopped = rs.assembler.is_stopped();
+        if stopped {
```

Comment:
Is just `stopped` enough to remove it, or should it be "dropped by application". One can call `stop` without dropping the stream. The next interaction would then lead to an unknown stream error?

Reply:
IMO this is sufficient. There's no operation that's meaningful to perform on a stopped stream. The documentation for each `pub fn stop` says: `Once stopped, a stream cannot be read from any further.` Perhaps we should make this clearer.